Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+click = "*"
+submitter = {editable = true, path = "."}
```

Comment:
Does that go in Pipfile? I dislike the way it currently works and will change this if I just drop that into Pipfile. (I'll deal with the Dockerfile changes this may cause)

Reply:
Yes! You can look at hoard or workflow for an example (this also won't work exactly as I wrote it because the current cli command isn't `main`, it's `hello`, but this is the general idea).

Reply:
It works! thx!

Reply:
For future us: while this works great, it does complicate the Docker builds as it only works in Pipenv environments as it is a feature of Pipenv. Installing via wheel (I don't know what to call that) as a multistage build seems to work, but it's still a bit unclear to me if this is "better" than having less complicated Docker builds.